middleware/capacity.go

```diff
@@ -0,0 +1,20 @@
+package middleware
+
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/livepeer/catalyst-api/pipeline"
+)
+
+const MAX_JOBS_IN_FLIGHT = 10
```

Comment:
I think Alex had thought of something shorter, maybe only 3. Wdyt?

Reply:
I think we can definitely handle more than 3, will bring it down to 5 to begin with